lib/bookmarker/cli.ex

```diff
@@ -16,6 +17,7 @@ defmodule Bookmarker.CLI do
       h: :help,
       f: :file,
       t: :title,
+      or: :order,
```

Comment:
I think it would be best to leave a letter only version '-o' instead of '-or'.

Reply:
I chose the use or as the char "o" was already being used for the "output" key  in line 24.

Reply:
Oh. My bad.

Maybe we could rename the flag to "--sort", "-s".

Reply:
Great! I'll fix that in a sec :+1: